makeflow/jx-args.json.example

```diff
@@ -0,0 +1,15 @@
+{
+  "ARCHIVE_URL": "https://de.cyverse.org/dl/d/00F871F7-800D-4E84-B1C0-3B15BB9DB2F4/stereoTop-2018-05-07__16-58-55-097.zip",
```

Comment:
Is there a script available that can generate this file?

Reply:
No, not yet.